### PR TITLE
Range expressions grammar

### PR DESCRIPTION
**Pull request by brauliobz**
Part of #84.

### CODE REVIEW

File: src/expressions/range-expr.md [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+> _RangeFullExpr_ :  
+> &nbsp;&nbsp; `..`  
+
 The `..` operator will construct an object of one of the `std::ops::Range` (or
-`core::ops::Range`) variants.
+`core::ops::Range`) variants, according to the following table:
+
+| Production             | Syntax        | Type                         | Range                 |
+|------------------------|---------------|------------------------------|-----------------------|
+| _RangeExpr_            | start`..`end  | [std::ops::Range]            | start &le; x &lt; end |
```
Comment (Havvy) [inline src/expressions/range-expr.md]: Might be worth using &lt;pre&gt; tags to preserve the formatting in the Range column?

**Comment by Havvy** [comment]
💟 Thanks!

File: src/expressions/range-expr.md (hunk 1 above)
Comment (brauliobz) [reply src/expressions/range-expr.md]: Do you mean to keep the &le;, x and &lt; aligned (like they are in the source code)?

Comment (Havvy) [reply src/expressions/range-expr.md]: Yes.

**Pull request merged**